### PR TITLE
Add structure

### PR DESCRIPTION
**Pull request by liranringel**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,6 +1,12 @@
 use clap;
 use std::env;
 
+
+mod config;
+mod iptables;
```
Comment (sandleru) [inline src/main.rs]: More explicit name would be better
Any ideas?

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
```
Comment (sandleru) [inline .idea/misc.xml]: Shouldn't it be ignorned?

File: .idea/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (guybortnikov) [inline .idea/.gitignore]: I prefer all the `.idea/` dir will get inside the `.gitignore`, I don't like any IDE-specific files inside repo.

Comment (guybortnikov) [reply .idea/.gitignore]: At least new line at end of file

Comment (liranringel) [reply .idea/.gitignore]: Added it

**Review by sandleru — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -1 +1,2 @@
 /target
+.idea/
```
Comment (guybortnikov) [inline .gitignore:2]: new line EOF

**Pull request merged**